test/unittests/test_parse_en.py

```diff
@@ -290,6 +290,17 @@ def test_combinations(self):
 
 
 class TestExtractNumber(unittest.TestCase):
+    def test_extract_number_decimal_markers(self):
```

Comment:
Should include a test case for `1,000,000` and `1,000.0` (or similar). Maybe also `"1000, is what I said"` and/or `"I said 1,000."` to include boundary cases